vars/withFortifySecrets.groovy

```diff
@@ -0,0 +1,15 @@
+def call(String product, String environment, Closure block) {
+  def fortifySecrets = [
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'fortify-on-demand-username', version: '', envVariable: 'FORTIFY_USER_NAME'],
+    [$class: 'AzureKeyVaultSecret', secretType: 'Secret', name: 'fortify-on-demand-password', version: '', envVariable: 'FORTIFY_PASSWORD'],
+  ]
+
+  String theKeyVaultUrl = "https://${product}-${environment}.vault.azure.net/"
```

Comment:
Are these secrets specific to a team ? Why not just put it in the Jenkins vault?

Reply:
Yes they are. Each team will use their own Fortify secrets.

Reply:
Not sure if every team has a shared key vault with `${product}-${environment}` though.

Reply:
We request this in the document. So, they will create 2 secrets on Azure Key Vaults with their Fortify creds. @MSancaktutar

Reply:
I echo what @adusumillipraveen says - we shouldn't assume or require that all products have a vault with this naming convention. Coupled with the comment above about passing around the environment for non-environment-specific tests it would be better to pass in the vault name and let users decide.

Reply:
I will have a look this.

Reply:
This is done and has been tested in user-profile. Now each client is able to pass its own keyvaultname. 

https://build.platform.hmcts.net/view/CDM/job/HMCTS_Nightly_CDM/job/ccd-user-profile-api/job/nightly-fod-scan-ccd-1257/